js/Scene3D.js

```diff
@@ -502,6 +502,22 @@ Scene3D.prototype = Object.create(EventSource.prototype, {
             offset_dimension = this._model_exploding.offset_dimension
             exploding_dimension = this._model_exploding.dimension
 
+            delta_x = model_max_x  - model_min_x;
+            delta_y = model_max_y  - model_min_y;
+            delta_z = model_max_z  - model_min_z;
+
+            var zPartitionSize = delta_z / number_of_partitions;
+            var zPartitionGap = delta_z * partition_separation_ratio / 2;
+            var yPartitionOffset = delta_y * slice_offsets / 2;
+
+            this.spots.forEach(function (spot) {
```

Comment:
yep, they're a bit off in dimensions of explosion and shift. the deviation grows from the bottom to top of the model